### PR TITLE
net-misc/electrum: add dev-lang/python-3.6 support

### PR DESCRIPTION
**Pull request by XVilka**

### CODE REVIEW

**Comment by XVilka** [comment]
See bug https://bugs.gentoo.org/641778

**Comment by SoapGentoo** [comment]
@mgorny `LINGUAS` hackery ahoy, what do we do? remove all of the `LINGUAS` hacks?

File: net-misc/electrum/electrum-3.0.3.ebuild [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 EAPI="6"
 
-PYTHON_COMPAT=( python3_{4,5} )
+PYTHON_COMPAT=( python3_{4,5,6} )
```
Comment (SoapGentoo) [inline net-misc/electrum/electrum-3.0.3.ebuild]: this ebuild has a number of issues, like depending on naked `dev-lang/python`, which should use `${PYTHON_DEPS}` instead

File: net-misc/electrum/electrum-3.0.3.ebuild [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 PYTHON_REQ_USE="ncurses?"
```
Comment (SoapGentoo) [inline net-misc/electrum/electrum-3.0.3.ebuild]: add EAPI-4 default specifier here `(-)`

File: net-misc/electrum/electrum-3.0.3.ebuild [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
 
 inherit distutils-r1 gnome2-utils xdg-utils
```
Comment (SoapGentoo) [inline net-misc/electrum/electrum-3.0.3.ebuild]: `doicon` requires `desktop.eclass`

**Comment by XVilka** [comment]
I updated it as requested

**Comment by XVilka** [comment]
I saw electrum was just bumped to 3.0.5. Is there any decision on this one?

**Pull request closed without merging**